packages/pds-ember/app/styles/pds/components/index.scss

```diff
@@ -1,12 +1,12 @@
 @use "app";
+@use "avatar";
 @use "banner";
 @use "breadcrumbs";
 @use "button";
 @use "button-set";
 @use "checkbox-field";
 @use "code-block";
 @use "copyright";
-@use "cta-link";
```

Comment:
was this removal an accident?

Reply:
Nope. CtaLink styles are just Button styles.